_posts/2015-12-10-RGSoC.markdown

```diff
@@ -0,0 +1,65 @@
+
+Together with [Boris Bügling](https://twitter.com/neonacho) we paired on creating [cocoapods-label](https://rubygems.org/gems/cocoapods-label), a plug-in for CocoaPods that annotates the Podfile with its Pod description. This is great for everyone with short memory that wants to remember what the different pods do in an efficient way. We started working on this while Boris visited San Francisco, during AltConf, and then, we continued working on it remotely, while screen sharing. We found that the best time to work on it was during Boris's evenings which were our mornings due to a 9 hour time difference. It was great to have Boris guide us and show us how to create a new plug-in, a knowledge we would have use for in the project.
+
+When we applied to work on CocaPods we choose to work on making CocoaPods more modular by creating plug-ins. During a planning meeting we picked issues that were interesting for us to work on.
+
+I (Emma) chose to work on a couple of different things. Starting with what seemed like an easy task, to make so that [Pods are listed case-insensitive in Xcode](https://github.com/CocoaPods/Xcodeproj/pull/294). I learned a lot about sorting and also about just finding my way around the codebase. To get that PR merged filled me with pride and energy to take on the next task. Continuing with moving the source code for [cocoapods-search](https://github.com/CocoaPods/cocoapods-search) to its own gem and depend on it from CocoaPods it self. There is a scaffold for [creating plugins](https://github.com/CocoaPods/cocoapods-plugin-template) for CocoaPods that I used that sets up the main structure of files and folders. From there on it came down to figuring out what code was responsible for search and copy that code to the newly created gem. I ran my local changes with Rake to make sure everything worked as it should. The tests also needed to be moved to the new gem and that proved to be more complicated than moving the source code, mostly since they required a lot of files in the fixture folder to run. One of the goals of extracting search to its own gem was to make it easier and safer to modify without messing something else up in the larger project. After I finished the extraction other contributors have continued to improve the search code even more which shows that the extraction did have a positive effect in that its easier to find and understand.
+
+With search finished there were still a couple of weeks left of the project. I wanted to learn more about web development and collaborated with [Orta](https://twitter.com/orta) on a new project called [CocoaPods Stories](https://github.com/CocoaPods/stories.cocoapods.org). The idea was to give Pod users the ability to write stories about what Pods they are using and how. We didn’t make it longer than the planning and research stage and we are not sure wether it will still be useful. Please ping us if you want us to continue.
```

Comment:
`wether` -> `whether` (typo)
`longer` -> `further` (further is generally used in these cases)
`Please ping us if you want us to continue` -> `Please ping us if you want to contribute to this idea` (call to action to do something, not to ask us to do something )